### PR TITLE
Add code splitting to TOO route

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Add code splitting to TOO route so that JS for the TOO part of the app isn't loaded until needed

## Reviewer Notes
- Replaced the client tests with an e2e instead because of added code splitting makes it difficult to test the existence of the route

## Code Review Verification Steps
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/168963802) for this change

### CODE REVIEW

File: src/scenes/Office/TOO/too.jsx [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import React from 'react';
+import PrivateRoute from 'shared/User/PrivateRoute';
 
 function TOO() {
   return <h1>TOO Placeholder Page</h1>;
 }
 
-export default TOO;
+function TOORoute() {
```
Comment (tinyels) [inline src/scenes/Office/TOO/too.jsx]: It's a little unusual to export a route. Did you need to do this to make the code-splitting work? If so, could you add some comments explaining that?

Comment (tinyels) [reply src/scenes/Office/TOO/too.jsx]: Though I wouldn't expect that to be necessary. See https://alligator.io/react/react-loadable/#route-based-code-splitting

Comment (Ronolibert) [reply src/scenes/Office/TOO/too.jsx]: I wasn't sure how else to code split while also taking advantage of our `PrivateRoute` component's auth checks

Comment (tinyels) [reply src/scenes/Office/TOO/too.jsx]: Is there something about our implementation of `PrivateRoute` that makes this not work? I would expect it to work the same way as `AddUser` does in the sample that I posted above.

Comment (Ronolibert) [reply src/scenes/Office/TOO/too.jsx]: I misunderstood!! Changed this now

**Comment by Ronolibert** [comment]
@tinyels Ready for re-review whenever you can!

File: src/scenes/Office/TOO/too.jsx (hunk 1 above)
Comment (tinyels) [reply src/scenes/Office/TOO/too.jsx]: Thanks!

**Pull request merged**